cmake/modules/Finddaxctl.cmake

```diff
@@ -0,0 +1,42 @@
+# - Find libdaxctl
+# Find the daxctl libraries and includes
+#
+# daxctl_INCLUDE_DIR - where to find libdaxctl.h etc.
+# daxctl_LIBRARIES - List of libraries when using daxctl.
+# daxctl_FOUND - True if daxctl found.
+
+find_path(daxctl_INCLUDE_DIR ndctl/libdaxctl.h)
```

Comment:
Should this be `daxctl/libdaxctl.h`?

Reply:
This is not an mistake. Libdaxctl.h exists in both directories h. I'm not sure which is better. see https://ubuntu.pkgs.org/20.04/ubuntu-main-arm64/libdaxctl-dev_67-1_arm64.deb.html also shows two:

```
Path
/usr/include/daxctl/libdaxctl.h
/usr/include/ndctl/libdaxctl.h
/usr/lib/aarch64-linux-gnu/pkgconfig/libdaxctl.pc
/usr/share/doc/libdaxctl-dev/copyright
```

Reply:
i am inclined to find `daxctl/libdaxctl.h` instead. for two reasons

- it's less confusing
- and more importantly, it's more portable. see https://centos.pkgs.org/8/centos-appstream-x86_64/daxctl-devel-71.1-2.el8.i686.rpm.html